test/deploy.sh

```diff
@@ -234,7 +234,7 @@ if [ "$DOTEST" == "true" ]; then
       i=$((i+1))
    done
    echo "waiting for the result"
-   sleep 20
```

Comment:
why sleep more?

Reply:
I simply found it is not enough to finish the cx, that makes the checking result failed.